wrapper.c

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * This file is part of nvme-cli
+ *
+ * Copyright (c) 2022 Daniel Wagner, SUSE
+ */
+
+#include <dlfcn.h>
+
+#include <libnvme.h>
+
+const char * __attribute__((weak)) nvme_get_version(enum nvme_version type)
+{
+	const char *(*libnvme_get_version)(enum nvme_version type);
+
+	libnvme_get_version = dlsym(RTLD_NEXT, "nvme_get_version");
```

Comment:
I am still pondering if this is a good idea. For one, building nvme-cli statically isn't possible with this anymore. At least not out of the box.

Reply:
After a bit of experimenting this isn't an issue. So even if we have this code in place, when the linker kicks in (for the shared version or the static version) it will always drop this weak function in favor of the one in libnvme if found. 

So we don't have to any fancy stuff here.